### DISCUSSION TITLE
posts/chi-sports-roundup-11-23

### DISCUSSION THREAD

**Discussion by giscus** [afreidz/afreidz.codes#3] [Blog Post Comments]
# posts/chi-sports-roundup-11-23

A semi-regular segment summarizing all the happenings in the Chicago sports landscape, because nothing rustles my jimmies like Chicago sports!

https://www.afreidz.codes/posts/chi-sports-roundup-11-23

**Comment by LaurisF** [comment]
I think Rossy was doing OK, not great, but as a mgr with no experiewnce, he learned and pushed some to be better. I agree, that if this move, a bold one at that is to be worthy, there better be some significant spending this off season. 

Fluss is a failed DC to HC move. Getsy rode the AR coat tails. If QB1 is to be developed, it is not by these two.

**Reply by afreidz** [reply]
It was encouraging to hear the press conference this morning, and Rossy's first public comments.  Seems like there will be no public bloodshed over this move.  Which helps.

RE: Fluss/Getsy ... 💯 agree that I don't want either of those two homunculi responsible for bringing up another QB1.

**Comment by LaurisF** [comment]
Haven't heard what happened at the presser, looking forward to hearing about it tonite.
________________________________
